### PR TITLE
Add io.kavan.GUADEC

### PR DESCRIPTION
**Pull request by kavanmevada**

### CODE REVIEW

File: io.kavan.GUADEC.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
+{
+    "app-id": "io.kavan.GUADEC",
+    "runtime": "org.gnome.Sdk",
+    "runtime-version": "3.36",
```
Comment (bilelmoussaoui) [inline io.kavan.GUADEC.json:4]:
```suggestion
    "runtime-version": "3.38",
```

File: io.kavan.GUADEC.json [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        {
+            "name" : "guadec-events",
+            "buildsystem" : "simple",
+            "build-commands": [
+		        "cargo pkg install /app"
+            ],
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://gitlab.gnome.org/kavanmevada/guadec-events.git"
```
Comment (nedrichards) [inline io.kavan.GUADEC.json:71]: this needs to be pointing to a release or a specific commit.

**Comment by nedrichards** [comment]
bot, build io.kavan.GUADEC

File: io.kavan.GUADEC.json [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    "runtime-version": "3.36",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "guadec-events",
+     "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "build-args" : [
+            "--share=network"
```
Comment (barthalion) [inline io.kavan.GUADEC.json:13]: We do not allow Internet access during build; you can generate a manifest for dependencies with https://github.com/flatpak/flatpak-builder-tools/tree/master/cargo.

**Comment by barthalion** [comment]
bot, build io.kavan.GUADEC

**Comment by barthalion** [comment]
bot, build io.kavan.GUADEC

File: io.kavan.GUADEC.json [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+                    "url" : "https://gitlab.gnome.org/GNOME/libhandy.git",
+                    "tag" : "1.0.0"
+                }
+            ]
+        },
+        {
+            "name" : "cargo-pkg",
+            "buildsystem" : "simple",
+            "build-commands": [
+	            "cargo install cargo-pkg --root /app"
```
Comment (barthalion) [inline io.kavan.GUADEC.json:51]: Is it needed for anything now when `cargo-sources.json` pulls in all dependencies?

File: io.kavan.GUADEC.json [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+                "/app/.crates.toml",
+                "/app/.crates2.json",
+                "/app/bin/cargo-pkg"
+            ]
+        },
+        {
+            "name" : "guadec-events",
+            "buildsystem" : "simple",
+            "build-commands": [
+		        "cargo pkg install /app"
```
Comment (barthalion) [inline io.kavan.GUADEC.json:63]: That's how other apps do it with cargo: https://github.com/flathub/com.github.jms55.Sandbox/blob/master/com.github.jms55.Sandbox.json#L26

**Comment by bilelmoussaoui** [comment]
bot, build io.kavan.GUADEC

**Comment by bilelmoussaoui** [comment]
bot, build io.kavan.GUADEC

**Comment by kavanmevada** [comment]
bot, build io.kavan.GUADEC

**Comment by kavanmevada** [comment]
bot, build io.kavan.GUADEC

**Comment by kavanmevada** [comment]
bot, build io.kavan.GUADEC

**Comment by kavanmevada** [comment]
bot, build io.kavan.GUADEC

**Comment by kavanmevada** [comment]
bot, build io.kavan.GUADEC

**Comment by kavanmevada** [comment]
bot, build io.kavan.GUADEC

**Comment by kavanmevada** [comment]
bot, build io.kavan.GUADEC

**Comment by kavanmevada** [comment]
bot, build io.kavan.GUADEC

**Comment by bilelmoussaoui** [comment]
bot, build io.kavan.GUADEC

**Comment by bilelmoussaoui** [comment]
bot, build io.kavan.GUADEC

**Comment by bilelmoussaoui** [comment]
bot, build io.kavan.GUADEC

**Pull request closed without merging**